src/index.js

```diff
@@ -113,6 +115,9 @@ export { GenericAd               } from './molecules/GenericAd';
 export { StickyAd                } from './molecules/StickyAd';
 export { LazyProgressiveImage    } from './molecules/LazyProgressiveImage';
 export { Source                  } from './molecules/Source';
+export { DetailsSection          } from './molecules/MainRecipe/DetailsSection';
+export { IngredientsSection      } from './molecules/MainRecipe/IngredientsSection';
```

Comment:
This does not return a `<section>`, might be slightly misleading. How about just `Ingredients`?